OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.47
+31 Oct 2025
+
+## Core
+* Added `alert_instance_id` configuration option to 'email.conf', 'hipchat_posturl.conf', and 'posturl.conf' files. When set to `True`, the instance ID of the OpsCenter sending the alert is included in the notification. (OPSC-17695)
+
+## UI
+* Added a new button in alert UI to allow users to test email and posturl alert notifications. (OPSC-17695)
```

Comment:
This description is incorrect. The ui change was to allow the users to specify the alert level to use when sending the all clear notification. So instead maybe "Added the ability to specify the alert level for the all clear notification when the alert is resolved.